pages/morse.js

```diff
@@ -71,7 +72,9 @@ export default function ButtonAppBar() {
                 <Typography variant="h5">tečka</Typography>
               </Button>
               <Button variant="outlined" onClick={handleSeparatorClick}>
```

Comment:
maybe add className={styles.characterButton} instead of className={morse_styles.symbol_button} at Typography